sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/implementation/models/CpkInfo.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.file.datalake.implementation.models;
+
+import com.azure.core.annotation.Fluent;
```

Comment:
Preferably remove these files that aren't actually a part of this feature. This'll get cleaned if you merge main back into your branch now that all 82 features are in (though you'll probably have to resolve some conflicts)